model/SimpleLicensing/Classes/AnyLicenseInfo.md

```diff
@@ -12,7 +12,8 @@ according to the SPDX license expression syntax.
 
 An AnyLicenseInfo is used by licensing properties of software artifacts.
 It can be a NoneLicense, a NoAssertionLicense,
-single license (either on the SPDX License List or a custom-defined license);
+single license (either on the [SPDX License List](https://spdx.org/licenses/)
+or [a custom-defined license](https://spdx.github.io/spdx-spec/v3.0/model/ExpandedLicensing/Classes/CustomLicense/));
```

Comment:
Links should never include version number, since we don't want to update them continuously. (and we're at 3.0.1 now).

Please always use relative links.

Reply:
Thank you. Will fix all instances to relative ones.